drivers/gles3/shaders/ycbcr.glsl

```diff
@@ -0,0 +1,52 @@
+[vertex]
+
+		layout(location = 0) in highp vec4 vertex_attrib;
```

Comment:
Extra tab or something here

Reply:
This is my copy of clang-format that keeps messing it up, should be resolved now that I've merged the logic into copy.glsl